terraform/modules/bosh_vpc/network_private.tf

```diff
@@ -48,31 +47,58 @@ resource "aws_route_table" "az2_private_route_table" {
   }
 }
 
+resource "aws_route_table_association" "az1_private_rta" {
+  subnet_id = "${aws_subnet.az1_private.id}"
+  route_table_id = "${aws_route_table.az1_private_route_table.id}"
+}
+
+resource "aws_route_table_association" "az2_private_rta" {
+  subnet_id = "${aws_subnet.az2_private.id}"
+  route_table_id = "${aws_route_table.az2_private_route_table.id}"
```

Comment:
The first time this is run, it's still using the older NAT instances, so shouldn't be any disruption. Once the newer instances are created, then we can go back and update the routes. This is why I documented this bit in the comments about how to rotate. New instances declared first, then update routes and EIP instances.